gocat/util/utility.go

```
@@ -64,3 +64,8 @@ func removeWhiteSpace(input string) string {
 		return r
 	}, input)
 }
+
+func StopProcess(pid int) {
+	proc, _ := os.FindProcess(pid)
+	_ = proc.Kill()
```

remove _ variable, since it is unused.

---

it is unused but is included to explicitly ignore the error of a failed process kill in the same convention as used other places (see https://github.com/mitre/sandcat/blob/master/gocat/util/utility.go#L25)